go/systests/teams_test.go

```diff
@@ -783,16 +783,20 @@ func (u *userPlusDevice) perUserKeyUpgrade() {
 }
 
 func kickTeamRekeyd(g *libkb.GlobalContext, t libkb.TestingTB) {
+	const workTimeSec = 10 // team_rekeyd delay before retrying job if it wasn't finished.
```

Comment:
make it `1`?

Reply:
ok I'm convinced - that's what serverside has been doing, in kind of sneaky way though which I completely missed - it doesn't call accelerate_team_rekeyd, but instead call set_config directly

Reply:
Done, changed to 3 sec